text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+
+Projects in the sandbox should leave it within one year, either by being merged into existing SIGs, being accepted as independent SIGs, or by being archived.
+
+This OTEP is inspired by [CNCF’s sandbox projects](https://www.cncf.io/sandbox-projects/), but the process is significantly different.
+
+## Internal details
+
+### Acceptance criteria
+
+A low barrier to entry would be desired for the sandbox. While the process can be refined based on our experience, my initial proposal for the process is the following:
+
+1. Proposals should be written following the template below and have one Technical Committee (TC) and/or Governance Committee (GC) sponsor, who will regularly provide the TC and GC information about the state of the project.
+1. Once a sponsor is found, the TC and GC will vote on accepting this new project on the Slack channel #opentelemetry-gc-tc. 
+    1. After one week, the voting closes automatically, with the proposal being accepted if it has received at least one :thumbs-up: (that of the sponsor, presumably).
+    1. If at least one :thumbs-down: is given, or a TC/GC member has restrictions about the project but hasn’t given a :thumbs-down:, the voting continues until a majority is reached or the restrictions are cleared.
+    1. The voting closes automatically once a simple majority of the TC/GC electorate has chosen one side.
+1. Proponents should abide by OpenTelemetry’s Code of Conduct (currently the same as CNCF’s).
+1. There’s no expectation that small sandbox projects will have regular calls, but there is an expectation that all decisions will be made in public and transparently.
+1. Sandbox projects do NOT have the right to feature OpenTelemetry’s name on their websites. 
+
+#### Template
+
+> Project name:
+> 
+> Repository name:
+>  
+> Motivation:
+> 
+> Zoom room requested?
+
+##### Example
+
+> Project name: OpenTelemetry Collector Community Distributions
+> 
+> Repository name: opentelemetry-collector-distributions
+>  
+> Motivation: The OpenTelemetry Collector Builder allows people to create their own distributions, and while the OpenTelemetry Collector project has no intentions (yet) on hosting other more specialized distributions, some community members are interested in providing those distributions, along with best practices on building and managing such distributions, especially around the CI/CD requirements.
+> 
+> Zoom room requested? No
+
+### Periodic reports
+
+On the first [GC/TC joint call](https://docs.google.com/document/d/1jylE5uZCKV9mrPw8Qrc5ExGyRVbBdqcbWPwni-hB5dE) of the calendar quarter (Jan, Apr, Jul, Oct), the TC/GC sponsors for the sandbox projects MUST provide an update about their sponsored project. Should the sponsor not be able to join the call, a short written report MUST be provided as part of the meeting notes.
```

Comment:
👎

Reply:
Is this a no for the periodic reports as a whole? Or for specific parts of it? Do you have any suggestions on how to keep track of the health of the projects? My concern is that projects might get accepted and forgotten there until it's time to pull the plug. Perhaps they need a nudge from time to time?

Note that I didn't state that sponsors should write the report themselves. They are on the hook for providing those reports: they can be written by sandbox members and validated by the sponsor before being relayed to the bigger group.

Reply:
I answered above https://github.com/open-telemetry/oteps/pull/233/files#r1224666071

Reply:
I'm marking this as resolved, as the discussion is happening on the issue you linked in the previous comment.